integration_tests/tests/forseti/controls/explain.rb

```diff
@@ -259,6 +259,25 @@
             end
         end
 
+        describe "List permissions in storage role" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer access_by_authz --permission iam.serviceAccounts.get").stdout).to match /roles\/editor/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer access_by_authz --permission iam.serviceAccounts.get").stdout).to match /project\/release-automate-silver/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer access_by_authz --permission iam.serviceAccounts.get").stdout).to match /serviceaccount\/project-factory-22907@release-automate-silver.iam.gserviceaccount.com/
```

Comment:
Is the service account here always in place for this test to pass? i.e. if the env is purged and repopulated, will this service account still be present?

Reply:
Yes, it will be in place.